### ISSUE TITLE
email from address non-configurable?

### ISSUE THREAD

**Issue by JohnnyPrimus** [ohadschn/letsencrypt-webapp-renewer#115]
Completely possible I'm missing something or just not understanding fully, but is there a way to configure the from address for the sendgrid notifications? I've created an apikey and verified a sender but it looks like its hard coded to using letsencrypt-webapp-renewer@ohadsoft.com as the from address... Is that accurate?

**Comment by ohadschn** [comment]
Correct, it's currently hard coded: https://github.com/ohadschn/letsencrypt-webapp-renewer/blob/ad566151841b6d9a2ce20d1e0c4e9572a07365ef/src/OhadSoft.AzureLetsEncrypt.Renewal.WebJob/Email/SendGridNotifier.cs#L47

Curious, why would you want to configure the from address?

**Comment by JohnnyPrimus** [comment]
I don't have a burning desire to configure it. Maybe I am misunderstanding sendgrid? The webapp logs show it succeeds in sending an email but I never receive the email, and sendgrid doesn't increase the count of sent emails. 

I had a suspicion that whatever was used as the from address needed to be a verified sender in sendgrid... is that not the case?

**Comment by ohadschn** [comment]
Not that I recall, maybe you could contact SendGrid support?

**Comment by JohnnyPrimus** [comment]
When I update the send address to an address that has been added to my sendgrid account as a verified sender, it completes. it seems to only want to send if the from address has been verified on that account....

**Comment by ohadschn** [comment]
You know that does ring a bell, but I set it up years ago and barely remember... 
Do you think you could add a short explanation to the README in a pull request?

**Comment by ohadschn** [comment]
`it seems to only want to send if the from address has been verified on that account....`
Not sure I understand that, my account doesn't have ohadsoft.com verified AFAIK.. are we talking about this?
![image](https://user-images.githubusercontent.com/8555641/97115348-fc941700-16fe-11eb-9a67-7c661acf2f99.png)

**Comment by JohnnyPrimus** [comment]
Do you have anything under single sender verification?

![image](https://user-images.githubusercontent.com/4716570/97116415-2e36bf00-16ba-11eb-8cd8-5927b677032a.png)

**Comment by JohnnyPrimus** [comment]
domain authentication gets rid of the "on behalf of" in the email (at least, in the gmail client)

but so far in my testing, single sender verification seems to be required for any email address you use as a from address

**Comment by ohadschn** [comment]
I can't find these settings in my account, maybe we have a different plan? I created a free/basic one via Azure a few years back.
Could you provide the URL of that settings page and/or explain/show screenshot how I get to it ?

**Comment by JohnnyPrimus** [comment]
Sure, the url is here:
https://app.sendgrid.com/settings/sender_auth

here is the docs about the single sender verification:
https://sendgrid.com/docs/for-developers/sending-email/sender-identity/


There are two ways you can confirm send identities:
1) Create a single sender and verify the account (by receiving email at it and clicking a verification link)
2) Proving your ownership of a domain by creating CNAME records at that domain

If you verify domain ownership, you can send as any account @thatdomain.com without needing to verify them individually, however the perms don't extend to other domains.

If you don't have a domain, or you want to send from a domain that you don't own, you must verify that you have access to that account 

Here's screencap I got after registering a new account and going to the sender page
![image](https://user-images.githubusercontent.com/4716570/97120915-e32ba480-16d7-11eb-89f3-781ec4252cf0.png)

**Comment by JohnnyPrimus** [comment]
This is a really great webapp (thanks), and I wanted to get it deployed to my site so I made a change to add fromEmail, and to add the documentation stuff you recommended.

Pr is here if you want to use any of it... (still learning so would love to learn if there are places I ought to have done something different)

Cheers!

**Comment by ohadschn** [comment]
I simply don't have those settings, I guess I'm on some legacy plan.
Thanks for the PR! will take a look.
